pallets/schemas/src/tests.rs

```diff
@@ -516,3 +518,70 @@ fn dsnp_broadcast() {
 	);
 	assert_ok!(result);
 }
+
+#[test]
+fn create_schema_with_settings_should_work() {
```

Comment:
Assuming the extrinsic isnt' supposed to be there any more, these tests can use `create_schema_via_governance`.

Reply:
If we can get away without creating a new extrinsic and only updating create_schema function then all good, though that's a breaking change 👀